src/machine/io.c

```diff
@@ -5,7 +5,7 @@
  *
  * Portions derived from musl:
  *
- * Copyright © 2005-2020 Rich Felker, et al.
+ * Copyright (c) 2005-2020 Rich Felker, et al.
```

Comment:
We should usually not touch external copyright headers. I think in this case it is acceptable.